sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/service/SettingService.java

```diff
@@ -0,0 +1,35 @@
+package com.gfg.sellercenter.setting.service;
+
+import com.gfg.sellercenter.setting.entity.Setting;
+import com.gfg.sellercenter.setting.reader.HttpReader;
+import lombok.AllArgsConstructor;
+import org.json.JSONObject;
+
+@AllArgsConstructor
+public class SettingService implements SettingServiceInterface {
+
+    private HttpReader httpReader;
+
+    public Setting getSetting(String field, int sellerId) throws Exception
```

Comment:
I would instead of overloading here use different method name to be more explicit, like `getSellerSetting()`